auto_py_to_exe/web/js/importExport.js

```diff
@@ -1,18 +1,20 @@
 const importConfiguration = (configuration) => {
     // TODO Check for version to support older versions
 
-    configuration.pyinstallerOptions.forEach(({ optionDest, value }) => {
-        if (configurationSetters.hasOwnProperty(optionDest)) {
-            configurationSetters[optionDest](value);
-        } else {
-            // TODO Warn user?
-            // TODO noconfirm is expected to come here
-        }
-    });
-
-    // setup nonPyinstallerOptions
-    recursionLimitToggle(configuration.nonPyinstallerOptions.increaseRecursionLimit);
-    document.getElementById('raw-arguments').value = configuration.nonPyinstallerOptions.manualArguments;
+    if ('pyinstallerOptions' in configuration) {
```

Comment:
Were you finding that `pyinstallerOptions` did not exist? `_collectDataToExport` looks like it will always provide it.
Is this check required? If it is, should we also be checking for `configuration.nonPyinstallerOptions`?

I see that you pass `{}` back from Python's `import_configuration` function - this is probably what the check is for. Why not keep the return as None if no configuration file is selected and check for null/undefined here (forgot which one it will be).

Reply:
In this case it's indeed better to keep `None`, I'll rollback these changes.

Reply:
Resolved!